ramls/groups.raml

```diff
@@ -1,6 +1,6 @@
 #%RAML 1.0
 title: User Groups
-version: v1.0.1
+version: v1.1.0
```

Comment:
Interface versions only have a a major and minor, please can this change this. It should also match the version in the module descriptor.

```suggestion
version: v15.3
```